testing/workflows/components/kfctl_test.jsonnet

```diff
@@ -230,6 +230,53 @@ local dagTemplates = [
     ),
     dependencies: ["kfctl-apply-gcp"],
   },
+  {
+    template: buildTemplate(
+      "install-spark-operator",
```

Comment:
So here in your test you are running ks directly to install the package so are you using deploy_kubeflow.py or test_deploy.py?

Reply:
Yeah I can revert those if we want, I figured it made sense to see Spark installed on minikube even if we only used the operator on the full version in e2e workflow.